source/apiVolontaria/volunteer/admin.py

```diff
@@ -2,8 +2,14 @@
 
 from . import models
 
+
+class ParticipationAdmin(admin.ModelAdmin):
+    search_fields = ['user__username']
+    list_display = ['user', 'standby', 'start_date', 'end_date', 'cell']
```

Comment:
It would have been cool to: 
- add filtering by `start_date`, `end_date` and `cell`
- list_display `user__fristname` and `user__lastname`, since username are not always clear.